_docs/overview/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Forseti conceptual overview
+order: 001
+hide: 
+  left_sidebar: true
+---
+
+# {{ page.title }}
+
+Forseti Security is a collection of community-driven, open-source tools to help
+you improve the security of your Google Cloud Platform (GCP) environments.
+Forseti includes core modules that you can enable, configure, and execute
+independently of each other. Community contributors can also develop add-on
+modules to offer unique capabilities. Forseti's core modules work together to
+provide their respective features, and provide a foundation that addons can
+build upon.
+
+Get started with
+[Forseti Security]({% link _docs/quickstarts/forseti-security/index.html %}).
+
+## When to use Forseti Security
+
+Use Forseti Security when you want to monitor your GCP resources to ensure that
+role-based access controls are set as you intended. Forseti Security can notify
+you if policies change unexpectedly and work automatically to keep your access
+policies in a known state. By using Forseti Security's modules together, you can
+make sure that there isn't any unauthorized access to your GCP resources.
+
+## How Forseti Security works
+
+When you install Forseti Security, you can configure and deploy core Forseti
+Security modules. The core Forseti Security modules work together to take a
+snapshot of your GCP resources, monitor those resources, and notify you of
+access policy changes.
+
+![Forseti architecture diagram](/images/docs/overview/diagram.png)
```

Comment:
Two things.

1) This overview only applies to 2.0 (Q1-2018). We either need to remove it or note as such.
2) We should reference the (#) annotations in the image with details.

Reply:
I'll remove it for now and work offline on annotation notes for when 2.0 is ready.